client/my-sites/plans/wx-medium/index.tsx

```diff
@@ -0,0 +1,142 @@
+	currentPlan: SitePlanData;
+}
+
+const WXMediumPlansPage = ( { currentPlan }: WXMediumPlansPageProps ) => {
+	const translate = useTranslate();
+
+	const annualPlan = getPlans()[ PLAN_WOOEXPRESS_MEDIUM ];
+	const monthlyPlan = getPlans()[ PLAN_WOOEXPRESS_MEDIUM_MONTHLY ];
+
+	const planPrices = useSelector( ( state ) => ( {
+		annualPlanPrice: getPlanRawPrice( state, annualPlan.getProductId(), false ) || 0,
+		annualPlanMonthlyPrice: getPlanRawPrice( state, annualPlan.getProductId(), true ) || 0,
+		monthlyPlanPrice: getPlanRawPrice( state, monthlyPlan.getProductId() ) || 0,
+		currencyCode: getPlan( state, annualPlan.getProductId() )?.currency_code || '',
+	} ) );
+
+	const isAnnualSubscription = currentPlan.productSlug === PLAN_WOOEXPRESS_MEDIUM;
+	const activePlan = isAnnualSubscription ? annualPlan : monthlyPlan;
+
+	const wooExpressMediumPlanFeatureSets = useMemo( () => {
+		return getWooExpressMediumFeatureSets( { translate } );
+	}, [ translate ] );
+
+	const monthlyPriceWrapper = <span className="wx-medium-plans__price-card-value" />;
+	const priceDescription = <span className="wx-medium-plans__price-card-interval" />;
+
+	const priceContent = isAnnualSubscription
+		? translate(
+				'{{monthlyPriceWrapper}}%(monthlyPrice)s{{/monthlyPriceWrapper}} {{priceDescription}}per month, %(annualPrice)s billed annually{{/priceDescription}}',
+				{
+					args: {
+						monthlyPrice: formatCurrency(
+							planPrices.annualPlanMonthlyPrice,
+							planPrices.currencyCode,
+							{ stripZeros: true }
+						),
+						annualPrice: formatCurrency( planPrices.annualPlanPrice, planPrices.currencyCode, {
+							stripZeros: true,
+						} ),
+					},
+					components: {
+						monthlyPriceWrapper,
+						priceDescription,
+					},
+				}
+		  )
+		: translate(
+				'{{monthlyPriceWrapper}}%(monthlyPrice)s{{/monthlyPriceWrapper}} {{priceDescription}}per month{{/priceDescription}}',
+				{
+					args: {
+						monthlyPrice: formatCurrency( planPrices.monthlyPlanPrice, planPrices.currencyCode, {
+							stripZeros: true,
+						} ),
+					},
+					components: {
+						monthlyPriceWrapper,
+						priceDescription,
+					},
+				}
+		  );
+
+	return (
+		<>
+			<BodySectionCssClass bodyClass={ [ 'is-woo-express-medium-plan' ] } />
+
+			<Card className="wx-medium-plans__price-card">
+				<div className="wx-medium-plans__price-card-text">
+					<span className="wx-medium-plans__price-card-label">{ translate( 'My Plan' ) }</span>
+					<span className="wx-medium-plans__price-card-title">{ activePlan.getTitle() }</span>
+					<span className="wx-medium-plans__price-card-subtitle">
+						{ translate( 'Accelerate your growth with advanced features.' ) }
+					</span>
+				</div>
+				<div className="wx-medium-plans__price-card-conditions">{ priceContent }</div>
+				<div className="wx-medium-plans__price-card-cta-wrapper">
+					<Button className="wx-medium-plans__price-card-cta">
+						{ translate( 'Manage my plan' ) }
+					</Button>
```

Comment:
This should take the user to the management page for the plan subscription. We may want to hide the button as a starting point if we don't have access to the purchase/subscription details.